apps/judicial-system/backend/src/app/modules/notification/services/subpoenaNotification/subpoenaNotification.service.ts

```diff
@@ -67,14 +67,14 @@ export class SubpoenaNotificationService extends BaseNotificationService {
     for (const recipient of to) {
       if (recipient.email && recipient.name) {
         promises.push(
-          this.sendEmail(
-            formattedSubject,
-            formattedBody,
-            recipient.name,
-            recipient.email,
-            undefined,
-            true,
-          ),
+          this.sendEmail({
+            subject: formattedSubject,
+            html: formattedBody,
+            recipientName: recipient.name,
+            recipientEmail: recipient.email,
+            attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.